### PR TITLE
Add Initial Manifests

### LINKED ISSUE #2: Populate This Repo With Input Manifests
## Background

We don't currently version inputs. They are too large to put in a `git` repository. @aidanheerdegen and myself were talking about having systems to backup these inputs by adding date-based subdirectories to tape storage (see https://github.com/ACCESS-NRI/model-config-inputs/issues/1#issuecomment-2372731534). This seemed pretty close to having a version control system. 

Instead of versioning the inputs themselves, we can version metadata about the inputs. This also gives a better place to house workflows and actions relating to metadata than the original `model-config-tests` repo. See https://github.com/ACCESS-NRI/model-config-inputs/issues/1

## Implementation Information

Create a directory structure similar to the one that we have on `Gadi` at `/g/data/vk83/configurations/inputs` except instead of the input files themselves, have a `manifest.yaml` file at the leaf of the directory tree that contains metadata about the given inputs. For example:

```
access-esm1p5
├── CHANGELOG
├── modern
│   ├── historical
│   │   ├── atmosphere
│   │   │   ├── aerosol
│   │   │   │   └── global.N96
│   │   │   │       └── 2021.06.22
│   │   │   │           └── manifest.yaml
│   │   │   ├── forcing
│   │   │   │   ├── global.N96
│   │   │   │   │   └── 2021.06.22
│   │   │   │   │       └── manifest.yaml
│   │   │   │   └── resolution_independent
│   │   │   │       └── 2021.06.22
│   │   │   │           └── manifest.yaml
│   │   │   └── land
...
access-om2
...
```

An example of manifests are found in https://github.com/ACCESS-NRI/access-om3-configs/tree/dev-025deg_jra55do_ryf/manifests - but we would probably look to do something slightly different. 

## TODOs

 - [x] Create an equivalent directory structure to `/g/data/vk83/configurations/inputs` in this repository, with the leafs being a single `manifest.yaml`. Consider just `git init`ing on `Gadi` and pushing the result to `main`, once the below is done. 
 - [x] Create a very restrictive `.gitignore` that only tracks the above

### PR DESCRIPTION
**Pull request by CodeGat**
This PR adds all the manifests generated in `/g/data/vk83/configurations/inputs` via https://github.com/ACCESS-NRI/model-config-inputs/blob/main/.github/scripts/generate_manifests.bash

Specifically, these are manifests from `access-om2`, `access-om3`, `access-esm1p5`, ~~`access-esm1p5-unstructured`~~ and `JRA-55`. 

Closes #2

### CODE REVIEW

**Comment by CodeGat** [comment]
Note - deleted all the manifests under `access-esm1p5-unstructured` as we are deleting that directory. 

Currently attempted to delete it using `tm70_ci`  but the following has stuck around (permission denied):
```
access-esm1p5-unstructured/
└── pre-industrial
    ├── atmosphere
    │   ├── CABLE-AUX-1.4
    │   │   └── core
    │   │       ├── biogeochem
    │   │       │   ├── modis_phenology_csiro.txt
    │   │       │   ├── pftlookup_csiro_v16_17tiles.csv
    │   │       │   ├── pftlookup_csiro_v16_17tiles_spinup.csv
    │   │       │   ├── pftlookup_csiro_v16_17tiles_wtlnds.csv
    │   │       │   └── poolcnpInTumbarumba.csv
    │   │       └── biogeophys
    │   │           ├── def_soil_params.txt
    │   │           └── def_veg_params.txt
    │   ├── stasets
    │   │   ├── X01001218
    │   │   ├── X01002207
    │   │   ├── X01003236
    │   │   ├── X01003237
    │   │   ├── X01003254
    │   │   ├── X01003255
    │   │   ├── X01003274
    │   │   ├── X01003275
    │   │   ├── X01003276
    │   │   ├── X01003277
    │   │   ├── X01003278
    │   │   ├── X01003279
    │   │   ├── X01003280
    │   │   ├── X01003281
    │   │   ├── X01003286
    │   │   ├── X01005207
    │   │   ├── X01005208
    │   │   ├── X01005222
    │   │   ├── X01005223
    │   │   └── X01010206
    │   └── STASHmaster
    │       ├── STASHmaster_A
    │       ├── STASHmaster_A.original
    │       ├── STASHmaster_O
    │       ├── STASHmaster_S
    │       ├── STASHmaster_W
    │       └── STASHsections_A
    ├── coupler
    │   ├── areas.nc
    │   ├── cf_name_table.txt
    │   ├── grids.nc
    │   ├── masks.nc
    │   ├── rmp_cice_to_um1t_CONSERV_FRACNNEI.nc
    │   ├── rmp_cice_to_um1u_CONSERV_FRACNNEI.nc
    │   ├── rmp_cice_to_um1v_CONSERV_FRACNNEI.nc
    │   ├── rmp_um1t_to_cice_CONSERV_DESTAREA.nc
    │   ├── rmp_um1t_to_cice_CONSERV_FRACNNEI.nc
    │   ├── rmp_um1u_to_cice_CONSERV_FRACNNEI.nc
    │   └── rmp_um1v_to_cice_CONSERV_FRACNNEI.nc
    └── ice
        ├── grid.nc
        ├── kmt.nc
        └── monthly_sstsss.nc
```
They will have to be deleted before this PR is merged.

**Comment by CodeGat** [comment]
They've been deleted!

**Comment by CodeGat** [comment]
Also deleted manifests from `JRA-55/JRA55-do-1-5-0-1` and `/g/data/vk83/configurations/inputs/JRA-55/MRI-JRA55-do` as those directories are being deleted

File: access-om3/ww3/mod_def/global.1deg/2024.04.18/manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+format: yamanifest
+version: 1.0
+---
+manifest.yaml:
+  fullpath: /g/data/vk83/configurations/inputs/access-om3/ww3/mod_def/global.1deg/2024.04.18/manifest.yaml
+  hashes:
+    binhash: 506fe562f9b3f966d163261c3359a581
+    md5: bb031c32181b5fc574c9f7141bafd6c7
```
Comment (aidanheerdegen) [inline access-om3/ww3/mod_def/global.1deg/2024.04.18/manifest.yaml]: Seem to have added an entry for the manifest itself.

File: access-om3/ww3/initial_conditions/global.1deg/2024.04.18/manifest.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+format: yamanifest
+version: 1.0
+---
+manifest.yaml:
+  fullpath: /g/data/vk83/configurations/inputs/access-om3/ww3/initial_conditions/global.1deg/2024.04.18/manifest.yaml
+  hashes:
+    binhash: cb53e27e650ef677702f0f51dabaedf4
+    md5: e7275dce634b60a74b859d4a0f9217db
```
Comment (aidanheerdegen) [inline access-om3/ww3/initial_conditions/global.1deg/2024.04.18/manifest.yaml]: Same here.

File: access-om3/wombat/initial_conditions/global.1deg/2024.04.02/manifest.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+init_ocean_wombatlite.res.nc:
+  fullpath: /g/data/vk83/configurations/inputs/access-om3/wombat/initial_conditions/global.1deg/2024.04.02/init_ocean_wombatlite.res.nc
+  hashes:
+    binhash: d90040503452b58e11de5af77f5d8c2c
+    md5: 1648193a63179513a705541d5f5f508f
+manifest.yaml:
+  fullpath: /g/data/vk83/configurations/inputs/access-om3/wombat/initial_conditions/global.1deg/2024.04.02/manifest.yaml
+  hashes:
+    binhash: 391e3b40d23a581db3d28c51f639a7e1
+    md5: ba48160143a98e75cc02f8d1d3daf93d
```
Comment (aidanheerdegen) [inline access-om3/wombat/initial_conditions/global.1deg/2024.04.02/manifest.yaml]: And here.

**Review by aidanheerdegen — CHANGES_REQUESTED** [review]
Seems some manifests have an entry for themselves.

I've pointed out a few, but not done an exhaustive search.

File: access-om3/ww3/mod_def/global.1deg/2024.04.18/manifest.yaml (hunk 1 above)
Comment (CodeGat) [reply access-om3/ww3/mod_def/global.1deg/2024.04.18/manifest.yaml]: Fixed via https://github.com/ACCESS-NRI/model-config-inputs/pull/6/commits/efad4afd534eb76f9fe3331bbdb62d11646babd7

File: access-om3/ww3/initial_conditions/global.1deg/2024.04.18/manifest.yaml (hunk 2 above)
Comment (CodeGat) [reply access-om3/ww3/initial_conditions/global.1deg/2024.04.18/manifest.yaml]: Fixed via https://github.com/ACCESS-NRI/model-config-inputs/pull/6/commits/efad4afd534eb76f9fe3331bbdb62d11646babd7

File: access-om3/wombat/initial_conditions/global.1deg/2024.04.02/manifest.yaml (hunk 3 above)
Comment (CodeGat) [reply access-om3/wombat/initial_conditions/global.1deg/2024.04.02/manifest.yaml]: Fixed via https://github.com/ACCESS-NRI/model-config-inputs/pull/6/commits/efad4afd534eb76f9fe3331bbdb62d11646babd7

**Comment by CodeGat** [comment]
Hey @aidanheerdegen, thanks for pointing that out. It was due to an initial half-run of `generate_manifests.yaml` which I cancelled and then rerun. The find command must have included the old `manifest.yaml` files in the new manifest! Fixed now :)

**Review by aidanheerdegen — APPROVED** [review]
LGTM.

**Pull request merged**